analyses/create-subset-files/create_subset_files.sh

```diff
@@ -40,18 +40,26 @@ Rscript --vanilla 02-subset_files.R \
 # histologies file
 cp $FULL_DIRECTORY/pbta-histologies.tsv $SUBSET_DIRECTORY
 
+# recurrently fused genes by histologies file
+cp $FULL_DIRECTORY/pbta-fusion-recurrently-fused-genes-byhistology.tsv $SUBSET_DIRECTORY
```

Comment:
It feels like there should be a hyphen between `by` and `histology` but if this is the file name, then this is what it is.